web_scraper/web_scraper/spiders/cve_spiders.py

```diff
@@ -0,0 +1,92 @@
+import scrapy
+import os
+
+
+class QuotesSpider(scrapy.Spider):
+    name = "cve" ## Spider name should be unique
```

Comment:
Maybe something more descriptive such as `ubuntu_cve_page_spider`? There are different types of CVEs we are likely going to deal with in the future, so we may also have something like `amazon_linux_cve_page_spider`, `pypi_cve_page_spider` etc.

Reply:
This is a very good thing to do. Will do it.